pkg/logql/shardmapper_test.go

```diff
@@ -782,6 +782,67 @@ func TestMapping(t *testing.T) {
 				},
 			},
 		},
+		{
+			in: `vector(0) or sum (rate({foo="bar"}[5m]))`,
```

Comment:
This test seems to be failing.
https://drone.grafana.net/grafana/loki/15228/2/5

Reply:
The original testcase is indeed wrong, and the 'Group' field is not considered.
 I just submitted a commit to fix this testcase